README.md

```diff
@@ -33,6 +33,10 @@ Presto has a comprehensive set of unit tests that can take several minutes to ru
 
     ./mvnw clean install -DskipTests
 
+Presto uses the [Provisio](https://github.com/jvanzyl/provisio) plugin for packaging using hard links. If your filesystem doesn't support hard links, you can bypass hard links using:
+    
+    ./mvnw clean install -presto.hard-link-includes=''
+
```

Comment:
My sense is that the vast majority of filesystems folks use do support hardlinks.  If that's the case, this is a very prominent callout in the README for relatively esoteric information.  I would remove it, and perhaps add a comment in the POM instead.

Reply:
IMO - Anyone setting up project & building it, is going to look in README file for build instructions.

Reply:
Maybe let's create a section toward the bottom for common issues with the build.  For now, we can just put this there, but add other stuff as needed (incorrect JDK, Maven timeouts, etc).

Reply:
Updated